x-pack/plugins/upgrade_assistant/public/application/components/overview/fix_issues_step/fix_issues_step.tsx

```diff
@@ -65,7 +102,14 @@ export const getFixIssuesStep = ({
           <p>
             <FormattedMessage
               id="xpack.upgradeAssistant.overview.fixIssuesStepDescription"
-              defaultMessage="Update your Elasticsearch and Kibana deployments to be compatible with Elastic 8.x. Critical issues must be resolved before you upgrade. Warning issues can be ignored at your discretion."
+              defaultMessage="Update your Elasticsearch and Kibana deployments to be compatible with Elastic 8.x. Critical issues must be resolved before you upgrade. Warning issues can be ignored at your discretion. {accessDeprecationLogsMessage}"
```

Comment:
This appears elsewhere, too, but 8.x makes me twitch because that's historically meant an unreleased version. (With much confusion around what the .x version meant.) Any chance we can use 8.n instead?

```suggestion
              defaultMessage="You must resolve any critical Elasticsearch and Kibana configuration issues before upgrading to Elastic 8.n. Ignoring warnings might result in differences in behavior after you upgrade. {accessDeprecationLogsMessage}"
```

Reply:
`8.n` sounds a bit strange for me and makes me wonder if it will confuse users too, what do you think about perhaps just phrasing it as 8? 

Note: we also phrase it as `8.x` in other parts of upgrade assistant, so if we do end up renaming it we will have to do so in other places too for consistency.

Reply:
@debadair let us know what's best. I agree that `8.x` is used in other places and if we change it here we'd need to update the other places too.

Reply:
Yeah, it's probably better to leave it as-is.

Reply:
I've updated the suggestion--I still think it's worth adjusting the wording.

Reply:
I like the change. Thanks @debadair ! 👍